src/scss/layout/_header.scss

```diff
@@ -1,3 +1,61 @@
-.header {
-  height: 90px;
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
 }
+
+body{
+    max-width: 1440px;
+    margin: auto;
+}
+
+header {
```

Comment:
you will have conflicts after merge your PR about header...
As recommendation next time, if you will work on a new feature, ALWAYS create your feature branch from `develop`.

Reply:
Done.